engine/src/cubos/engine/transform/plugin.cpp

```diff
@@ -0,0 +1,41 @@
+#include <cubos/core/settings.hpp>
+
+#include <cubos/engine/transform/local_to_world.hpp>
+#include <cubos/engine/transform/plugin.hpp>
+#include <cubos/engine/transform/position.hpp>
+#include <cubos/engine/transform/rotation.hpp>
+#include <cubos/engine/transform/scale.hpp>
+
+using namespace cubos;
+
+static void applyTransform(
+    core::ecs::Query<engine::LocalToWorld&, const engine::Position*, const engine::Rotation*, const engine::Scale*>
```

Comment:
Unrelated to the PR but:
```suggestion
using cubos::core::ecs::Query;
using namespace cubos::engine;

static void applyTransform(
    Query<LocalToWorld&, const Position*, const Rotation*, const Scale*>
```